### PR TITLE
Lra 152 add maintenance mode setup

### PR DESCRIPTION
**Pull request by britaumich**
I added the turnout gem and stole the maintenance page from your other app.
The notification table/process/display is stolen from MMSS camp notes.

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -18,11 +18,11 @@ default: &default
 development:
```
Comment (rsmoke) [inline config/database.yml:18]: Did you intend to push this database.yml config too?

**Review by rsmoke — COMMENTED** [review]
maybe not update the database.yml

What happened to my maintenance message I sent you yesterday

File: config/database.yml (hunk 1 above)
Comment (britaumich) [reply config/database.yml]: No, of course not. I restore the database.yml.

**Comment by britaumich** [comment]
Your maintenance message was lost.

**Review by rsmoke — APPROVED** [review]

**Review by rsmoke — APPROVED** [review]

**Review by rsmoke — APPROVED** [review]

**Pull request merged**